### ISSUE TITLE
The Bottom Menu is a bit strange

### ISSUE THREAD

**Issue by ooade** [mykeels/live-band#1]
- Initially, I didn't even know there was a bottom menu
- Active Menus does not look active when active :sweat_smile: i.e Piano seem active while Drums is active and Vice Versa.
- Jam doesn't seem to work. I haven't gone through the code tho.

BTW, Nice project. I got to know about `express-ws`. Dived into that :smile_cat:

**Comment by mykeels** [comment]
Thanks, Demola. 

Where would you suggest the menu be?

Jam isn't available as a feature yet, so it should probably be taken away.

Yep, I like express-ws cos its abstraction fits into express' pipeline smoothly

**Comment by ooade** [comment]
The Menu could be on top or may be fixed to the bottom. The latter may not play well with the Piano. Besides, the current position seems to be the best option for the Piano :sweat_smile:

**Comment by mykeels** [comment]
How about indicators that point users to the bottom when they first load the app?

**Comment by ooade** [comment]
@mykeels Yeah. That's awesome :fire: :fire: :fire:

**Comment by mykeels** [comment]
Here's what the bottom menu should look like

![image](https://user-images.githubusercontent.com/11996508/46920047-27134b80-cfe0-11e8-8782-4755c7857dd2.png)

- The white border color signifies the tab is "active"
- The "Jam" menu item is removed till it's available

Still no updates on the indicator that tells users about the menu at the bottom. I don't even know what it's normally called 😄Maybe make a separate issue for that, if it's necessary?

**Comment by ooade** [comment]
Yeah. LGTM!
I don't know what that indicator is called too :sweat_smile:
Hey @whizkydee, WDYT?

**Comment by whizkydee** [comment]
Great job on this, @mykeels!

Just checked out the app and I was quite confused too about which button was active. I'm in support of [this](https://github.com/mykeels/live-band/issues/1#issuecomment-429646109) implementation though – easy to understand.

@ooade, an indicator is an indicator, no special term for that. I'm not sure though; what kind of indicator are y'all talking about?

**Comment by ooade** [comment]
@whizkydee Indicator, like an arrow pointing down immediately the component get rendered to let the user know there's a menu bar below.

**Comment by whizkydee** [comment]
Oh yeah, totally. Actually, the menu should be at the top. It's much more accessible there.

I also think you can reduce the height of the menu and strip off the additional `<span>` below those buttons. It's totally irrelevant. I'm sure that was just a quick copy-paste thing, @mykeels -- understandable.

I think we can forget about the indicator once this is done. Good luck!
